.pipelines/build.yaml

```diff
@@ -25,6 +25,18 @@ jobs:
         displayName: "Set up the Go workspace"
 
       - script: |
+          # Static binary workaround for CodeQL and Go 1.21 and higher
```

Comment:
Can we please add this link as comment for future refernece : https://eng.ms/docs/cloud-ai-platform/devdiv/one-engineering-system-1es/1es-docs/codeql/troubleshooting/onboarding/language-compiled